packages/terra-framework-site/src/app/common/menu/MenuToolbar.scss

```diff
@@ -0,0 +1,52 @@
+@import '~terra-mixins';
+
+.terraClinical-MenuToolbar {
+  color: #1C1F21;
+  font-size: 14px;
```

Comment:
Update to `1rem`, you may be able to delete font-size as it will inherit the font-size set on html which computes to 14px.